dist/src/parser-engine.js

```diff
@@ -97,11 +99,17 @@ class ParserEngine {
         }
         this.appRoot = path.resolve(this.projectPath, this.projectOptions.baseUrl);
         this.appRoot = path.resolve(this.appRoot, this.projectOptions.outDir);
-        let fileList = new Array();
-        this.walkSync(this.appRoot, fileList, ".js");
-        for (var i = 0; i < fileList.length; i++) {
-            let filename = fileList[i];
-            this.processFile(filename);
+        let JSfileList = new Array();
+        this.walkSync(this.appRoot, JSfileList, ".js");
+        for (var i = 0; i < JSfileList.length; i++) {
+            let filename = JSfileList[i];
+            this.processJSFile(filename);
+        }
+        let TSFileList = new Array();
+        this.walkSync(this.appRoot, TSFileList, ".ts");
```

Comment:
how about `tsx` ?

Reply:
We can add `tsx`, too. Are there any difference with ```ts``` import / export statements, do you have a sample?